server/db/db.js

```diff
@@ -0,0 +1,54 @@
+const knex = require('knex')
+const config = require('./knexfile').development
```

Comment:
Let's rename this file and model the knex config after the other existing db modules.